sorachung/StoriesByUs
# View User Profiles

#### Changes Made

1. Added file `Profile.js, Dashboard.js, ProfileBookmarksList.js, ProfileStoriesList.js` to `client/src/components/profiles/2. Added file `filename` to `directoryname` directory.
3. Added file `MyProfile.js, MyDashboard.js,  MyProfileBookmarksList.js, MyProfileStoriesList.js` to `client/src/components/myProfile/` directory.

1. Modified file `StoryController.cs, StoryRepository.cs, IStoryRepository.cs, storyManager.js` to include `get stories by user id` and functionality.
2. Modified file `BookmarkController.cs, BookmarkRepository.cs, IBookmarkRepository.cs, bookmarkManager.js` to include `get bookmarks with story info by user id` and functionality.
3. Modified file `Bookmark.cs` to include `Story property` and functionality.
4. Modified file `UserController.cs, UserRepository.cs, IUserRepository.cs, userManager.js` to include `getting current user endpoint and methods` and functionality.
5. Modified file `Navbar.js` to include `updated links to my profile menu options` and functionality.
6. Modified file `filename` to include `functionalityname` and functionality.
7. Modified file `filename` to include `functionalityname` and functionality.
8. Modified file `filename` to include `functionalityname` and functionality.
9. Modified file `filename` to include `functionalityname` and functionality.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout src-profiles
   ```
10. Open a new Terminal tab (⌘T) and navigate to the server directory.
11. Test app functionality.
   > Go to browse all
   > on one of the stories, click on the user's display name
   > click between dashboard, stories, and bookmarks and verify that everything shows up and works
12. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.

closes #21
